style.css

```diff
@@ -1,12 +1,15 @@
+@import url("https://fonts.googleapis.com/css2?family=Poppins&display=swap");
+
 * {
   box-sizing: border-box;
   margin: 0;
   padding: 0;
 }
 
 body {
-  font-family: sans-serif;
   width: 375px;
+  background-color: #e5e5e5;
+  font-family: Poppins, Helvetica, sans-serif;
```

Comment:
- [ ] Please know that the width here shouldn't be fixed, as your webpage should be responsive for mobile screens that have a width range of `375px` to at least `768px`. 😁

- [ ] Kindly remove and adapt any fixed `width` you used in your styling, to adapt to larger mobile screens. 😁